ui/desktop/electron-app/src/services/client-daemon-manager.js

```diff
@@ -91,7 +94,8 @@ class ClientDaemonManager {
     return unixSocketRequest(request, postBody);
   }
 
-  search(requestData) {
+  async search(requestData) {
+    const start = Date.now();
     if (isWindows()) {
       return searchCliCommand(requestData);
```

Comment:
im assuming adding logging for searchCliCommand will be done in a later PR ?

Reply:
Hmm this one encompasses `searchCliCommand` so it shouldn't be necessary?